pages/posts/flight-of-a-byte.tsx

```diff
@@ -0,0 +1,120 @@
+                </p>
+                <h3>Decrypting and finishing</h3>
+                <p>
+                We&apos;re almost done! When the “Hello” message is fetched, it&apos;s placed in the local copy of the message log in the correct order. (Messages in the device&apos;s storage remain encrypted.) This internally sends the <code>EventReplicated</code> event which is received (<a href="https://github.com/berty/weshnet/blob/aa44f7d64dcbc54c643cfddfaac8aa0a4d80f6a0/store_message.go#L452" target="_blank" rel="noreferrer">code</a>) by the Wesh message monitor. We started at the libp2p “level” which doesn&apos;t care about the meaning of pubsub messages, up through the message log level which only cares about the correct order of messages. Now at the “top” level, Wesh must use <code>client1&apos;s</code> cryptographic keys to decrypt (<a href="https://github.com/berty/weshnet/blob/aa44f7d64dcbc54c643cfddfaac8aa0a4d80f6a0/store_message.go#L139" target="_blank" rel="noreferrer">code</a>) the message.
+                </p>
+                <p>
+                The decrypted message is sent internally (<a href="https://github.com/berty/weshnet/blob/aa44f7d64dcbc54c643cfddfaac8aa0a4d80f6a0/store_message.go#L195" target="_blank" rel="noreferrer">code</a>) to all monitoring processes, including the <code>GroupMessageList</code> service (<a href="https://github.com/berty/weshnet/blob/aa44f7d64dcbc54c643cfddfaac8aa0a4d80f6a0/api_event.go#L191" target="_blank" rel="noreferrer">code</a>). The <code>client1</code> application has called <code>GroupMessageList</code>, so the “Hello” message is sent (<a href="https://github.com/berty/weshnet/blob/aa44f7d64dcbc54c643cfddfaac8aa0a4d80f6a0/api_event.go#L199" target="_blank" rel="noreferrer">code</a>) to the application which prints the message.
+                </p>
+                <h3>Flight of a Byte</h3>
+                <p>
+                In summary, the flight of a message from <code>client2</code> to <code>client1</code> starts when the <code>client2</code> peer is discovered, subscribes to the libp2p pubsub topic for the group messages and publishes its message log head. <code>client1</code> doesn&apos;t have this head yet, so it uses the CID to fetch the message using IPFS. When the message arrives, it is added to <code>client1&apos;s</code> copy of the message log in the correct order. Wesh sends this new message to processes using the <code>GroupMessageList</code> service, such as the sample application which receives it and prints “Hello”. Easy, right? Each of these steps is needed to make sure communication can still happen even if a peer disconnects or reconnects at a different network address.
```

Comment:
CID was in code blocks everywhere but here.


```suggestion
                In summary, the flight of a message from <code>client2</code> to <code>client1</code> starts when the <code>client2</code> peer is discovered, subscribes to the libp2p pubsub topic for the group messages and publishes its message log head. <code>client1</code> doesn&apos;t have this head yet, so it uses the <code>CID</code> to fetch the message using IPFS. When the message arrives, it is added to <code>client1&apos;s</code> copy of the message log in the correct order. Wesh sends this new message to processes using the <code>GroupMessageList</code> service, such as the sample application which receives it and prints “Hello”. Easy, right? Each of these steps is needed to make sure communication can still happen even if a peer disconnects or reconnects at a different network address.
```